### DISCUSSION TITLE
Energy : If updated every day, there is no way to have a graph with default view

### DISCUSSION THREAD

**Discussion by benda95280** [home-assistant/frontend#20951] [Dashboard] [answered]
Dear team,

If your energy consumption is updated every day, you only have the previous day with data.
I haven't been able to found a way to have the previous day shown on graph with default view.
You can tick the box "Compare data", but choice is not saved for the next time.

Thanks!

**Comment by karwosts** [comment, marked as answer]
Implemented in https://github.com/home-assistant/frontend/pull/24654

**Comment by Spongman** [comment]
this saves the selection once one is made, but it doesn't really address the original issue, which is: which range to show the _first_ time it's visited. this case still shows 'today', which for people with delayed reporting is useless. there should be a way to specify the default range in the yaml.
